src/themes/simple/templates/Layout/BlogEntryPage.ss

```diff
@@ -1,19 +1,75 @@
 <!-- content-section -->
-<section class="content-section content-section-common $topClass" id="{$URLSegment}">
+<section class="content-section align-c content-section-common $topClass" id="{$URLSegment}">
     <div class="container about-content">
         <div class="flex-col-md-8 flex-col-md-offset-2">
             <h2 class="orange">$Title</h2>
-            $Thumbnail
-            <hr class="mr-t-50 mr-b-50">
-            <% if $Description %>
+            <p class="better-text">$Author</p>
+            <hr class="mr-t-5 mr-b-50">
+            <% if $Header %>
                 <div class="content intro-text">
-                    $Description
+                    $Header
                 </div>
                 <hr class="mr-t-50 mr-b-50">
             <% end_if %>
-            <div class="content better-text">
+            <div class="content better-text align-j">
                 $Content
+                <div class="info-box9">
+                    <h4>Share This Post: </h4>
+                    <div class="img social">
+                        <a href="https://www.facebook.com/sharer.php?u=$AbsoluteLink" target="_blank">
+                            <i class="fa fa-facebook-official"></i>
+                        </a>
+                        <a href="https://www.twitter.com/share?url=$AbsoluteLink" target="_blank">
+                            <i class="fa fa-twitter-square"></i>
+                        </a>
+                    </div>
+                </div>
+            </div>
+            <% if $ShowKeepReading %>
+                <hr class="mr-t-10 mr-b-30 bdr-dark1">
+                <h3 class="hd-3">Keep Reading</h3>
+                <div class="gt40">
+                    <div class="col-md-5" style="height: 253px;">
+                        <% if $PrevNextPage('prev') %>

```

Comment:
There's a pretty minor bug here - if there are only 2 articles on the site, this shows the other article as the next and previous article in the read more section.  So just jump from 1 to 3 articles!  You could look to see if they are the same, and only show one, but it's probably not worth the effort.

Reply:
Yea I decided to add that `<% if $KeepReading %>` statement a few lines above so you can handle that case in the CMS.